piyushgarg-dev/piyushgargdev-nextjs
# fix: color of social media while hovering

## What does this PR do?
This PR change the color of the social media those present in the nav bar while hovering


i just gone in the header component and change the text color of the social media 



Fixes #130



## Type of change


- This change requires a documentation update

## How should this be tested?
-[ ] go to the homepage nav bar section
-[ ] hover on the social media link img
-[ ] check the color while hovering 

## Mandatory Tasks

- [X ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist
